client/components/comments/commentFeed.tsx

```diff
@@ -0,0 +1,51 @@
+// src/components/CommentFeed.tsx
+import React, { useState, useEffect, use } from 'react';
+import Comment from '@/components/comments/comment';
+import { motion, AnimatePresence } from 'framer-motion';
+import { CommentType } from '../../../shared/types';
+
+interface CommentFeedProps {
+  comments: CommentType[];
+}
+
+const CommentFeed: React.FC<CommentFeedProps> = ({ comments }) => {
+  const [displayedComments, setDisplayedComments] = useState<CommentType[]>([]);
+  const [commentsIndex, setCommentsIndex] = useState(0);
+
+  useEffect(() => {
```

Comment:
small suggestion for the set interval, i think we'll want to use the `displayTime` field rather than a static number to make it a more natural flow(?)
instead of setInterval, we can possibly use `setTimeout` since each comment will be displayed for a different amount of time -- maybe in the first useEffect, we'll create a setTimeout, and then in the next useEffect another setTimeout?
![image](https://github.com/user-attachments/assets/6b809340-79cd-4766-91aa-968643f28c7b)

Reply:
will take a look at this in a later pr